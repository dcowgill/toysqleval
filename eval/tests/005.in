create table foo (
    bar varchar,
    baz number,
    qux integer,
    foo timestamp
);

insert into foo (qux, baz, bar, foo) values (42, -3.14159, 'hello, world', 'Sun May 10 21:39:19 -0400 2009');
insert into foo (baz, foo) values (0.0001, '1890-04-14T17:37:15-05:00');
insert into foo (qux, bar) values (99, 'goodbye, world');

select bar, qux, bar, foo, baz/0.5, baz, qux*2 from foo;
select bar||bar||bar from foo;
