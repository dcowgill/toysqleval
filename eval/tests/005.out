OK
OK
OK
OK
 bar              | qux | bar              | foo                       | ?        | baz      | ?  
------------------+-----+------------------+---------------------------+----------+----------+-----
 "hello, world"   | 42  | "hello, world"   | 2009-05-10T21:39:19-04:00 | -6.28318 | -3.14159 | 84 
                  |     |                  | 1890-04-14T17:37:15-05:00 | 0.0002   | 0.0001   |    
 "goodbye, world" | 99  | "goodbye, world" |                           |          |          | 198
 ?                                           
----------------------------------------------
 "hello, worldhello, worldhello, world"      
                                             
 "goodbye, worldgoodbye, worldgoodbye, world"
